src/components/Thanks/ThanksPageShare.vue

```diff
@@ -263,16 +276,21 @@ export default {
 				utm_campaign: this.utmCampaign,
 				utm_content: this.utmContent,
 			};
-			if (!this.calculatePeopleQtyToGoal()) {
+
+			if (!this.calculatePeopleQtyToGoal() && !this.isGuest) {
 				if (this.lender.public && this.lender.publicName) {
 					args.lender = this.lender.publicName;
 				}
 				args.funded_share = 1;
 				return getFullUrl(`${base}/invitedby/${this.lender.inviterName}`, args);
```

Comment:
@emuvente Should we have a check here for `this.lender.inviterName` before attempting to use the invitedby url? Same scenario is below...